lib/src/database/hive_collections_database.dart

```diff
@@ -1095,6 +1095,15 @@ class HiveCollectionsDatabase extends DatabaseApi {
     return;
   }
 
+  @override
+  Future<void> storeRoomAccountData(BasicRoomEvent event) async {
+    await _roomAccountDataBox.put(
+      TupleKey(event.roomId ?? '', event.type).toString(),
+      copyMap(event.toJson()),
```

Comment:
if I'm not wrong, we only need to use `copyMap` when we get something from DB and not to store

Reply:
Ah you are right. Thanks

Reply:
Okay actually only in the Matrix SDK database. In Hive I think we still need to create a copy before storing it. At least in the other parts we are still doing this